### ISSUE TITLE
3.2 Fintetune CNN

### ISSUE THREAD

**Issue by hughmancoder** [hughmancoder/ML-based-analysis-of-sound#2]
Fine-tune CNN to classify the 4 Chinese instruments in our dataset + 11 Western Instruments, totalling 15 classes
Evaluate validation accuracy and training metrics (refer to evaluation metrics in our existing train script)

Options
- [x] Train CNN from scratch on all 15 classes
- [x] Fine tune CNN on just the 4 new classes
